pycontw/pycontw-frontend
# Changes for generating full static site

- build(nuxt-generate): add dynamic routes configuration and update to use asyncData fetching data




## Types of changes


* **New feature**

## Description
- Update configuration to be able to generate dynamic routes
- Update to use asycData so that the pages could be full static. (I'm still figuring out the reason; the data only showed up in full static mode in this approach anyway.)

## Steps to Test This Pull Request
1. `npm run json-server`
2. `npm run generate`
3. `npm run start`
4. Check out if all pages works well.

## Expected behavior
- All pages should show up data as usual.

## Related Issue
(Not sure if it could fix_ #308)
Fix #339
